awsaws-ses/aws-ses/src/main/java/com/brother/awsses/dto/ReceiverInfo.java

```diff
@@ -0,0 +1,13 @@
+package com.brother.awsses.dto;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.List;
+
+@Setter
+@Getter
+public class ReceiverInfo {
```

Comment:
The class name is **ReceiverInfo** but found a variable whose name is *subject*.

### Suggestion
We can give the name of this class like EmailInfo, EmailTemplate, and add htmlContent variable into this class.

### Important
- We need to give the option for HTML and non-HTML flag
- We need to think/implement sending emails with attachments. We have to find this feature in AWS.